web/index.html

```diff
@@ -47,6 +47,20 @@
         ::-ms-reveal {
             display: none;
         }
+
+        /* At the time of writing this User Agent styles for autofill inputs in Chrome are not customizable. This hack exploits the fact that
+        these styles are applied with a transition. Chromium team has labeled this as a WontFix so it seems there's not much else we can do here. */
+        input:-webkit-autofill,
+        input:-webkit-autofill:hover,
+        input:-webkit-autofill:focus,
+        textarea:-webkit-autofill,
+        textarea:-webkit-autofill:hover,
+        textarea:-webkit-autofill:focus,
+        select:-webkit-autofill,
+        select:-webkit-autofill:hover,
+        select:-webkit-autofill:focus {
+            -webkit-transition-delay: 99999s;
```

Comment:
<img width="1047" alt="Screenshot 2022-11-29 at 1 37 39 PM" src="https://user-images.githubusercontent.com/83179295/204490458-67955e45-fd26-410b-a53d-861ad27092e4.png">

**NAB:** Can we also add `transition-delay`?

Reply:
+1

Reply:
Not sure if we need this, we are only targeting -webkit properties here.